applications/nrf_desktop/src/modules/dvfs.c

```diff
@@ -35,32 +35,49 @@ enum state {
 
 static enum state module_state = STATE_DISABLED;
 
+#define LIST_FOR_DVFS_STATE(MACRO, ...)						\
+	MACRO(INITIALIZING, __VA_ARGS__)					\
+	MACRO(LLPM_CONNECTED, __VA_ARGS__)					\
+	MACRO(USB_CONNECTED, __VA_ARGS__)					\
+	MACRO(CONFIG_CHANNEL, __VA_ARGS__)					\
+	MACRO(SMP_TRANSFER, __VA_ARGS__)					\
+	MACRO(COUNT, __VA_ARGS__)
+
+#define DEFINE_ENUM(name, ...) DVFS_STATE_ ## name,
```

Comment:
`DVFS_STATE(name, ...)`?

Reply:
We could reuse this macro later.

Reply:
Can't re use it as we also have "," in this macro which isn't used elsewhere

Reply:
Sorry, I missed it initially